quartz/plugins/transformers/badges.ts

```diff
@@ -0,0 +1,281 @@
+import { QuartzTransformerPlugin } from "../types"
+import badgesCSS from "../../styles/badges.scss"
+import { JSResource, CSSResource } from "../../util/resources"
+import icons from "lucide-static"
+
+export const BADGE_TYPES: any[] = [
```

Comment:
Could have a proper type, considering the structure is very clear. (array of arrays of ID, name, icon)